src/components/book-seat.js

```diff
@@ -69,6 +73,31 @@ const Booking = () => {
     // if not user, Route to the Booking page and load the signup modal
   };
 
+  // const getTrip = async (placeId) => {
+  //   const response = await fetch(`${BASE_URL}trip/${placeId}`);
+  //   let trips = await response.json();
+  //   trips = trips.data.map((tripData) => ({
+  //     ...tripData,
+  //     label: tripData.destination.name,
+  //     value: tripData.destination.id,
+  //   }));
+  //   setDestination(trips);
+  // };
```

Comment:
I'm trying to fire this endpoint call to populate the destination dropdown once a starting point is selected. I couldn't get it to work.

Please assistance is needed.
@DeolaJ